agent/acs/handler/acs_handler.go

```diff
@@ -69,6 +70,39 @@ type StartSessionArguments struct {
 	ECSClient            api.ECSClient
 	StateManager         statemanager.StateManager
 	AcceptInvalidCert    bool
+	_time                ttime.Time
```

Comment:
I'd like to punt on this for now.  I think there's value in reusing the logic, but the better value is switching to proper dependency injection and removing the logic altogether.